Makefile

```diff
@@ -75,3 +99,4 @@ down-pg_translit:
 clean: down-redis down-pg_translit down-pg
 	find . -name __pycache__ -exec rm -rf {} +
 	find . -name "*.pyc" -exec rm -rf {} +
+	rm compose-env/*.env
```

Comment:
This isn't what I would like: if I have defined own settings (ie. testing the production settings locally) then they will be removed.